lib/RequestRouter.js

```diff
@@ -92,20 +79,6 @@ RequestRouter.ERROR_UPLOAD_TOO_SMALL              = 'ERROR_UPLOAD_TOO_SMALL';
 RequestRouter.ERROR_UPLOAD_MIME_TYPE              = 'ERROR_UPLOAD_MIME_TYPE';
 RequestRouter.ERROR_UPLOAD_NOT_FOUND              = 'ERROR_UPLOAD_NOT_FOUND';
 
-var remove$Props = function(resource) {
```

Comment:
No because in validateCastParms we are ignoring the $ from validation ... this was initially written so that when we validate resource we don't check for the $ resources... but now validateCastParams take care of it+